packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -46,11 +43,8 @@ test.describe('Firefox Desktop Sync v3 sign up', () => {
     expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
   });
 
-  test('coppa disabled', async ({
-    target,
-    page,
-    pages: { login, connectAnotherDevice },
-  }) => {
+  test.skip('coppa disabled', async ({ target }) => {
```

Comment:
Are we filing a follow up for this one? If intentional, maybe mention in the PR comment that this was skipped?

Reply:
I just noticed the jira ticket specifies this should be skipped. I think the PR / commit message should note this though so that if someone is looking at git history they know it's on purpose.